c/tskit/tables.h

```diff
@@ -4217,7 +4217,9 @@ int tsk_table_collection_deduplicate_sites(
 int tsk_table_collection_compute_mutation_parents(
     tsk_table_collection_t *self, tsk_flags_t options);
 int tsk_table_collection_compute_mutation_times(
-    tsk_table_collection_t *self, double *random, tsk_flags_t TSK_UNUSED(options));
+    tsk_table_collection_t *self, double *random, tsk_flags_t options);
```

Comment:
whoops - you've un-marked tsk_table_collection_compute_mutation_times's options as unused (they are actually unused; I just checked)

Reply:
```suggestion
    tsk_table_collection_t *self, double *random, tsk_flags_t TSK_UNUSED(options));
```

Reply:
You don't need to do this in the header file though, it's just in the .c.